docs/data_quality.md

```diff
@@ -0,0 +1,114 @@
+# Properties of a High Quality OSV Record
+
+## Version
+
+1.0.0 (SEMVER)
+
+## Purpose
+
+Describe the “good enough” OSV record that will be imported by OSV.dev
+
+### Out of scope
+
+This does not discuss the problem of record bit rot over time, after initial successful import. The problem of continuous revalidation and treatment of records that have been successfully imported will be dealt with separately in .
+
+Deferred to a future iteration: validating the existence of vulnerable functions in the `ecosystem_specific` field, if supplied.
+
+## Audience
+
+1. Current and aspiring OSV record producers
+2. Downstream OSV.dev record consumers
+
+## Rationale
+
+OSV.dev seeks to be an comprehensive, accurate and timely database of known vulnerabilities (that is highly automation friendly). In order to meet this accuracy goal, a quality bar needs to be both defined and sustainably enforced.
+
+## Properties of a High Quality OSV Record
+
+### Valid
+
+As a prerequisite, it is assumed that a record passes [JSON Schema validation](#appendix-a-osv-schema-validation) for the version of the OSV Schema it declares itself to comply with in the `schema_version` field, or 1.0.0 if it does not.
+
+### Precise
+
+A high quality OSV record allows a consumer of that record to be able to answer the following questions in an **automated** way, at scale:
+
+* “Does this vulnerability, as described, impact me?
+  * “What version do I need to upgrade to for it not to impact me?” 
+
+The definition of “impact” will vary depending on how fine-grained the information available is (i.e. package-level or symbol-level for software library packages). Package-level precision is the minimum standard.
+
+#### Properties
+
+* for version and commit ranges
+  * `affected[]`.`ranges[]`.`introduced` is defined
+  * prefer `affected[]`.`ranges[]`.`fixed` over `affected[]`.`ranges[]`.`last_affected`
+    * this minimizes false negatives
+  * distinct ranges for `introduced..fixed` and/or `introduced..last_affected` *(i.e. introduced and fixed commits can't be the same)*
+  * values in `introduced` are before/less than `fixed`/`last_affected`
```

Comment:
This opens the door to the quagmire of "according to whom?" because of how bad humans are at labeling things in the absence of strictly enforced semantic versioning. Could we add some kind of qualifier here that covers the endless nuances of the potential answers? (Consider this a starting suggestion - I defer to others if there's a better/more succinct way to describe the authoritative source)

```suggestion
  * values in `introduced` are before/less than `fixed`/`last_affected` according to the canonical package registry
```